test/test_swizzle.cpp

```diff
@@ -651,4 +651,43 @@ TEST_F(SwizzleTest, TransformPropagatorSkipSwizzleOnTarget) {
   }));
 }
 
+TEST_F(SwizzleTest, SwizzleInRFactor) {
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+  auto tv0 = makeSymbolicTensor(2);
+  fusion->addInput(tv0);
+  auto tv1 = set(tv0);
+  tv1->split(1, 8);
+  tv1->split(0, 8);
+  tv1->reorder({{2, 1}});
+  tv1->swizzle(SwizzleType::XOR, 2, 3);
+  tv1->reorder({{2, 1}});
+  tv1->merge(0);
+  tv1->merge(1);
+  tv1->commitLeafToRFactor();
```

Comment:
I can not think of a better way than just create a "view-like" op and check the input and output.